task005/shape/circle/circle.go

```diff
@@ -0,0 +1,36 @@
+package circle
```

Comment:
It looks like you have not quite figured out the interfaces in GO. 
You have to implement Stringer interface. https://pkg.go.dev/fmt#Stringer
Any type that has a `String () string` method implements the Stringer interface. https://tour.golang.org/methods/17
So you have to add 
```
func (c Circle) String() string{
...
}

```
and test it accordingly.
The same about Rectangle.